README.md

```diff
@@ -44,7 +44,7 @@ branch.
 | `release/0.5` | [`draft-ietf-ppm-dap-04`][dap-04] | Yes | Unmaintained as of June 24, 2024 |
 | `release/0.6` | [`draft-ietf-ppm-dap-07`][dap-07] | Yes, [with errata](#draft-ietf-ppm-dap-07-errata) | Unmaintained as of June 24, 2024 |
 | `release/0.7` | [`draft-ietf-ppm-dap-09`][dap-09] | Yes, [with errata](#draft-ietf-ppm-dap-09-errata) | Supported |
-| `main` | [`draft-ietf-ppm-dap-14`][dap-14] | Yes | Supported |
+| `main` | [`draft-ietf-ppm-dap-15`][dap-15] | Yes | Supported |
```

Comment:
Question for the team here: if we think draft-ietf-ppm-dap-15 is done, and we expect to move on to implementing -16, then we might as well go ahead and cut `release/0.8` and then continue implementing -16 on `main`. But we can also wait if we want to land more stuff on `main` before branching.